lib/seahorn/Smt/TypeChecker.cc

```diff
@@ -12,10 +12,92 @@ namespace {
 
 //==-- main implementation goes here --==//
 class TCVR {
+  ExprMap m_cache;
+
+  bool isConst(Expr exp) {
+    return bind::isBoolConst(exp) || bind::isIntConst(exp);
+  }
+
+  bool isVar(Expr exp) { return bind::isBoolVar(exp) || bind::isIntVar(exp); }
```

Comment:
I think vars are deprecated. Don't worry about them. We have constants and bound variables, but no variables.